src/infra/cryptography/verification-token-adapter.ts

```diff
@@ -0,0 +1,7 @@
+import { type TokenGenerator } from '@/data/protocols/'
+
+export class VerificationTokenGenerator implements TokenGenerator {
+  async generate (): Promise<string> {
+    return Math.floor(100000 + Math.random() * 900000).toString()
```

Comment:
Is this going to be a promise?

Reply:
I think that's not a problem return a Promise. I'm just following the interface of TokenGenerator, but this implementation is not asynchronous.

Reply:
I understood her, okay then, in the future we'll think better about it.